File: TP_Blog/tools/DatabaseLinker.php
<?php

class DatabaseLinker
{

    private static $url = "mysql:host=localhost;dbname=tpBlogBDD;charset=utf8";
    private static $username = "root";
    private static $password = "";


    public static function getConnexion()
    {
        return new PDO(self::$url, self::$username, self::$password);
    }
}

File: JAVA/tp_qcm/src/main/java/com/utils/DatabaseLinker.java
package utils;

import java.sql.*;

public class DatabaseLinker {
    
    private static String url = "jdbc:mysql://localhost:3306/tpqcm?useSSL=false";
    private static String login = "root";
    private static String password = "root";
    private static Connection connex = null;
    
    
    public static Connection getConnexion() {
        try {
            connex = DriverManager.getConnection(url, login, password);
        } catch (SQLException ex) {
            ex.printStackTrace(System.err);
        }
        return connex;
    }
}
